cmdutil/closureLogic.go

```diff
@@ -0,0 +1,33 @@
+package cmdutil
+
+import (
+	"github.com/dachad/check-max-tcp-connections/tcpclient"
```

Comment:
imports are not properly sorted

Reply:
there's no "good order" for imports, but i left the ones from the stdlib at the beginning..